### DISCUSSION TITLE
Regarding the Addition of Support for Chinese Models

### DISCUSSION THREAD

**Discussion by xiaoliula** [immich-app/immich#12187] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Immich has significant growth potential in China, but currently, the smart search models do not support Chinese. I found a model under immich-app/XLM-Roberta-Large-Vit-B-16Plus, and I am unsure if this model supports Chinese. According to online descriptions, it is supposed to support Chinese. However, when I added it to Immich's smart search, I encountered errors during searches with the message: "ERROR: Exception in ASGI application." Additionally, a series of errors appeared in the immich_machine_learning container on Docker Desktop. I hope you can look into this request. Thank you.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by radh21301** [comment]
Hey. Can you post the content in English so the devs can help you out?

**Comment by bo0tzz** [comment]
Please post the actual error logs.

**Comment by pyccl** [comment]
XLM-Roberta-Large-Vit-B-16Plus这个是支持中文的，但是不太好。搜出来的前几个还好，越往后越不搭边。还是兴文的要好。我用的是windows 的docker desktop，机器学习文件在硬盘里，在yml文件中挂载到了/cache下。基本都正常吧。就是机器太老，学习时CPU100%。不到7000照片和视频，机器学习完一次需要不到10小时。

**Reply by mertalev** [reply]
Posting translation:
> XLM-Roberta-Large-Vit-B-16Plus supports Chinese, but it is not very good. The first few that came out were okay, but the ones that came out later were less and less relevant. Xingwen's is still better. I used docker desktop for windows, and the machine learning files were in the hard disk, which were mounted to /cache in the yml file. It was basically normal. But the machine was too old, and the CPU was 100% when learning. With less than 7,000 photos and videos, it took less than 10 hours for the machine to complete one machine learning.

Search doesn't filter irrelevant results; it just ranks your assets from most to least relevant. We hope to improve this in the future.

What is Xingwen (兴文)?
